source/databases-collections.txt

```diff
@@ -17,7 +17,7 @@ Navigate Your Data
 Once you :ref:`connect to your deployment <vsce-connect>` using |vsce|,
 use the left navigation to:
 
-- Explore your databases, collections, read-only views, and documents. 
+- Explore your databases, collections, read-only views, and documents.
```

Comment:
What about indexes? There is a subsection about indexes in this topic, and it does not surface in the TOC or titles shown in the TOC, just yet. Can indexes be added to this list?

Reply:
Agreed that we shouldn't fix there. I will add this to the ticket I opened.